### PR TITLE
Create stale.yml to automatically close old issues/PRs

### PR DESCRIPTION
**Pull request by keertk**

### CODE REVIEW

**Comment by keertk** [comment]
Hey @Wyverald could you please take a look? I've set `debug-only: true` for now.
This workflow is only for issues/PRs created this year.

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          This issue has been automatically closed due to inactivity. If you're still interested in pursuing this, please reach out 
+          to a maintainer and/or add the "not stale" label. Thanks!
+        stale-pr-message: >
+          Thank you for contributing to the Bazel repository!
+          This pull request has been marked as stale since it has not had any activity in the last 30 days. It will be closed in the next 
+          14 days unless any other activity occurs or one of the following labels are added: "not stale", "awaiting-review", "awaiting-PR-merge".
+          Please provide a status update or let us know when you're ready.
+        close-pr-message: >
+          This pull request has been automatically closed due to inactivity. If you're still interested in pursuing this, please reach 
+          out to a maintainer and/or add the "not stale" label. Thanks!
```
Comment (Wyverald) [inline .github/workflows/stale.yml:44]: same as above

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        stale-issue-label: 'stale'
+        exempt-issue-labels: 'not stale,awaiting-bazeler'
+        stale-pr-label: 'stale'
+        exempt-pr-labels: 'not stale,awaiting-review,awaiting-PR-merge'
+        exempt-draft-pr: true
+        operations-per-run: 500
+        ascending: true
+        start-date: 2023-01-01
+        debug-only: true
```
Comment (Wyverald) [inline .github/workflows/stale.yml:53]: do you want to keep this dry-run only?

Comment (Wyverald) [reply .github/workflows/stale.yml:53]: oh I just saw your message. Yeah this is fine for now

File: .github/workflows/stale.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        days-before-pr-stale: 30
+        days-before-issue-close : 7
+        days-before-pr-close: 14
+        stale-issue-message: >
+          Thank you for contributing to the Bazel repository! 
+          This issue has been marked as stale since it has not had any activity in the last 30 days. It will be closed in the next 7 
+          days unless any other activity occurs or one of the following labels are added: "not stale", "awaiting-bazeler".
+        close-issue-message: >
+          This issue has been automatically closed due to inactivity. If you're still interested in pursuing this, please reach out 
+          to a maintainer and/or add the "not stale" label. Thanks!
```
[the remark this answers is not in the document]
Comment (keertk) [reply .github/workflows/stale.yml]: Done

File: .github/workflows/stale.yml (hunk 1 above)
Comment (keertk) [reply .github/workflows/stale.yml]: Done

**Review by Wyverald — APPROVED** [review]

**Pull request closed without merging**